src/detail-page/components/tabs/Tab.tsx

```diff
@@ -11,23 +11,27 @@ const StyledTab = styled(MuiTab)`
   }
 `;
 
-interface TabProps {
+type TabProps = {
   text: string;
   chosenTab: number;
   index: number;
-  onClick: (index: number) => void;
-}
+  handleClick: (index: number) => void;
+} & MuiTabProps;
```

Comment:
We are making our custom `Tab` component, but under the hood it relies on Mui Tab. This is why we want to **preserve** Material-defined props (MuiTabProps) in addition to our custom ones (text, chosenTab, etc). To do this, we changed the type and we use the `...rest` operator to capture any props that we may pass to Tab in Tabs.tsx.

This gives us the following benefit: if in the future we want to modify the tab, we can simply use [Material API](https://material-ui.com/api/tab/). That is, we can pass a Material-defined prop (for example: `disableRipple` or `icon`) to _our_ Tab in Tabs.tsx and the magic of `...rest` will pass it to Material component (StyledTab).